### DISCUSSION TITLE
Real-time collaborative editing

### DISCUSSION THREAD

**Discussion by nexflo** [craftcms/cms#8527] [Ideas]
Multi-user auditing of the CMS should be possible. Ideally realtime google docs like editing, as a first step locking documents so one knows someone else is editing that entry.

**Comment by lukeholder** [comment]
Good idea. Check out https://github.com/marionnewlevant/craft-snitch in the meantime.

**Comment by brandonkelly** [comment]
Warning users when someone else is editing the entry is covered by #824.

**Comment by Kashkin** [comment]
I would love to see feature-parity with [Statamic](https://statamic.com/addons/statamic/collaboration).
